lab1.ipynb

```diff
@@ -0,0 +1,990 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(1, 1)\n",
+    "ax.hist(d,10, density=True)\n",
+    "plt.tick_params(labelsize = 12)\n",
+    "plt.xlim([-1,3.05])\n",
+    "x = np.linspace(-4,3.05,1000)\n",
+    "ax.plot(x,rayleigh.pdf(x,loc = 0, scale = 0.1),linewidth = 8,alpha = 0.7)\n",
+    "ax.set_xlim([-0.025,1])\n",
+    "ax.set_ylim([0,7])\n",
+    "plt.show()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0b3bba06-dd5c-4128-8a86-b3acd233944c",
+   "metadata": {},
+   "source": [
+    "## 3) Determining sigma value of my Rayleigh distribution"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9a14ceab-35ce-4b15-b3a3-4f7673b8a326",
+   "metadata": {},
+   "source": [
+    "### &nbsp;&nbsp;a) Hypothetical measurement"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3aa06d69-9779-4d4b-a52b-a36597f6e8d7",
+   "metadata": {},
+   "source": [
+    "####  &nbsp; &nbsp; &nbsp; &nbsp; choosing to be 0.02"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ca6e241e-c17d-4d87-b3c7-66026b705aa7",
+   "metadata": {},
+   "source": [
+    "### &nbsp;&nbsp;b) Statistical question"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0028c247-ba08-447f-8d00-ef9d4380bdb6",
+   "metadata": {},
+   "source": [
+    "####  &nbsp; &nbsp; &nbsp; &nbsp; What is the probability of recieving a signal of 3.02 or higher"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9114a97e-d03a-4b84-ba0a-752d8ae85eec",
+   "metadata": {},
+   "source": [
+    "### &nbsp;&nbsp;c) Mathematical integral"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 184,
+   "id": "312ec751-b98c-455a-b958-aefc655d40f2",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "theta = max(rayleigh.pdf(x,loc = 0.0, scale = 0.1))"
```

Comment:
I'm not sure how the library works for Rayleigh stuff, but this looks like it would have a mean of 0. I would just double check to see